models/User.js

```diff
@@ -63,13 +73,15 @@ module.exports = function(sequelize, DataTypes) {
     'id',
     'username',
   ]);
-  
+
   User.apiSettableFields = Object.freeze([
     'firstName',
     'lastName',
     'email'
   ]);
 
+  User.globalPermissions = PERMISSIONS;
```

Comment:
Is it possible to make `globalPermissions` a const to ensure it doesn't get accidentally reassigned (not sure if you can, just wondering)

Reply:
Should it be `GLOBAL_PERMISSIONS` because it's a constant value?